ui/main/src/app/modules/card/components/card-actions/card-actions.component.ts

```diff
@@ -81,9 +82,18 @@ export class CardActionsComponent implements OnChanges, OnDestroy {
             !this.isReadOnlyUser &&
             this.cardState.copyCardEnabledOnUserInterface &&
             this.cardState.userCard &&
+            this.doesTheUserHavePermissionToPublishCard() &&
             UserService.isWriteRightsForProcessAndState(this.card.process, this.card.state);
     }
 
+    private doesTheUserHavePermissionToPublishCard(): boolean {
+        if (!this.cardState.userCard.publisherList) return true;
+        const userEntities = UserService.getCurrentUserWithPerimeters().userData.entities;
+        const allowedPublishers = EntitiesService.resolveEntities(this.cardState.userCard.publisherList);
```

Comment:
i suggest to name the method isUserMemberOfAnEntityAllowedToPublishForThisState() 

This is to be more precise and avoid confusion with the other control :   UserService.isWriteRightsForProcessAndState

Reply:
done